src/erlcloud_aws.erl

```diff
@@ -144,6 +144,11 @@ aws_request4(Method, Protocol, Host, Port, Path, Params, Service, Config) ->
             {error, Reason}
     end.
 
+aws_request4_no_update(put, Protocol, Host, Port, Path, Params, Service, #aws_config{} = Config) ->
```

Comment:
Adding **put** to case clause didn't help much, I tried. Actually, this is "dirty" patch. The problem was not only the put was not supported, but also narrow semantic of Params, which in my case would be binary (json). I will try to rework this.